### DISCUSSION TITLE
Latest version is vulnerable?

### DISCUSSION THREAD

**Discussion by pixincreate** [bromite/bromite#1879] [General]
I'm on latest version of `Bromite (v99.0.4844.55)` while the Google Chrome is `v99. 0.4844.58`. Upon opening Bromite, I saw "exclamation mark"  on to right with on clicking redirected to [this page] (https://www.bromite.org/vulnerable-version). Screenshot attached below for reference.
![Screenshot_20220311-205534_Bromite](https://user-images.githubusercontent.com/69745008/157897487-acc32322-6c3a-4bc5-b70c-229310c8399c.png)

Bromite checks upstream with Beta or anything?

**Comment by csagan5** [comment]
It is a flaw in the update check logic; will be fixed on next release.
